contracts/Concil.sol

```diff
@@ -116,7 +116,7 @@ contract Concil is Ownable {
         ProposalInfo storage pInfo = proposalInfos[_id];
 
         // if lockedTime > 0 and lockedTime + lockTime < now, the proposal is still locked
-        require(pInfo.lockedTime == 0 || pInfo.lockedTime + lockTime < block.timestamp, "Proposal is locked");
+        // require(pInfo.lockedTime == 0 || pInfo.lockedTime + lockTime < block.timestamp, "Proposal is locked");
```

Comment:
why

Reply:
I don't know how to control mining in test, so this `require` always fails, `lockedTime + lockTime` always greater than `now`